stacks/nodejs/devfile.yaml

```diff
@@ -1,9 +1,9 @@
 schemaVersion: 2.1.0
 metadata:
   name: nodejs
-  version: 1.0.1
```

Comment:
Should't changing the underlying runtime version would be a semver-major bump?

Reply:
fixed